src/options.cpp

```diff
@@ -2206,9 +2206,15 @@ void options_manager::add_options_debug()
          true
        );
 
+    add( "MADE_OF_EXPLODIUM", "debug", translate_marker( "Made of explodium" ),
+         translate_marker( "If the number is higher than 0, explosive items and traps will detonate when hit by damage exceeding the threshold." ),
```

Comment:
```suggestion
         translate_marker( "Explosive items and traps will detonate when hit by damage exceeding the threshold.  A higher number means more damage is required to detonate.  Set to 0 to disable." ),
```
May help clarify how the setting works and make it clear to newer players that higher means less sensitive. Picked phrasing based off how evolution scaling (another setting where lower number means more dangerous, but zero means off) is worded.